Intersections_3/include/CGAL/Point_3_Iso_cuboid_3_intersection.h

```diff
@@ -0,0 +1,81 @@
+#ifndef CGAL_POINT_3_ISO_CUBOID_3_INTERSECTION_H
+#define CGAL_POINT_3_ISO_CUBOID_3_INTERSECTION_H
```

Comment:
I'm pretty sure that some header checking will break if this is not after the license header